android/src/main/java/com/pedrouid/crypto/RCTAes.java

```diff
@@ -126,7 +126,7 @@ private static String encrypt(String textBase64, String hexKey, String hexIv) th
 
         Cipher cipher = Cipher.getInstance(CIPHER_ALGORITHM);
         cipher.init(Cipher.ENCRYPT_MODE, secretKey, hexIv == null ? emptyIvSpec : new IvParameterSpec(Hex.decode(hexIv)));
-        byte [] textBytes = java.util.Base64.getDecoder().decode(textBase64);
+        byte [] textBytes = android.util.Base64.decode(textBase64, 0);
```

Comment:
In other places in the code we use `Base64.DEFAULT` instead of `0`.

Reply:
Please update